### PR TITLE
Fixed .hdr support to compile on windows

### PR DESCRIPTION
**Pull request by zer0problem**
Added "static_cast" when calling "pow" as the windows compiler threw an error about casting.
Edit: changed from "static_cast" to writing writing 2.0f and 128.0f instead
Edit2: squashed it
Edit3: changed commit msg

### CODE REVIEW

**Comment by reduz** [comment]
might probably be better to just write 2.0 and cast as float though :(

File: modules/hdr/image_loader_hdr.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ Error ImageLoaderHDR::load_image(Ref<Image> p_image, FileAccess *f) {
 		//convert
 		for (int i = 0; i < width * height; i++) {
 
-			float exp = pow(2, ptr[3] - 128);
+			float exp = pow( static_cast<float>(2), static_cast<float>(ptr[3] - 128) );
```
Comment (karroffel) [inline modules/hdr/image_loader_hdr.cpp]: why not make it `pow(2.0f, ptr[3] - 128.0f)`?

Comment (zer0problem) [inline modules/hdr/image_loader_hdr.cpp]: I changed it as per your request now. didn't think that far..

**Comment by akien-mga** [comment]
Could you squash the two commits together? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Review by karroffel — APPROVED** [review]

**Comment by vnen** [comment]
You should change the commit message too, since you changed the fix.

**Pull request merged**